packages/atomic/src/components/search/atomic-result/atomic-result.tsx

```diff
@@ -108,6 +113,11 @@ export class AtomicResult {
     event.detail(this.result);
   }
 
+  @Listen('atomic/resolveStopPropagation')
+  public resolveStopPropagation(event: CustomEvent) {
+    event.detail(this.stopPropagation);
+  }
```

Comment:
I feel like it's odd to have that property on the result instead of the link 🤔 

I'm thinking that maybe we should always stop propagation on links. I don't really see a case where we don't want to?

Reply:
I put it on `atomic-result` to make it more of an internal thing. 
If we don't propagate here, clicking on the result link won't work...

Reply:
this is what happens if I stop propagation in the result link
https://user-images.githubusercontent.com/30511433/174802196-998edb13-bcd6-459e-86d5-053cc5794f29.mp4

Reply:
not just to 'make it an internal thing' actually, but also because it simply doesn't work without it, so I don't want the user to be able to change it themselves

Reply:
Ok, so to summarise my explanation from stand up:
We currently STOP propagation from the link click.
My suggestion is to allow this event to be propagated so that we can use it in the `li` component and trigger `onSelect`. Otherwise we won't be able to click the link itself when trying to navigate..

And a quick explanation about the methods from [here](https://stackoverflow.com/a/5299841/8454914):
`stopPropagation` will prevent any **parent handlers** from being executed 
`stopImmediatePropagation` will prevent any **parent handlers** and also any **other handlers** from executing

Reply:
terrible naming if you ask me!!!!

Reply:
Well programmers are very good at naming things normally: https://en.wikipedia.org/wiki/HTTP_referer